.ci/containers/puppet-forge/Dockerfile

```diff
@@ -0,0 +1,64 @@
+from alpine:latest as resource
```

Comment:
Could we use the Ruby container we already have?

Reply:
Eh... we *could*, but I don't really want to.  The one we already have is go + ruby.  This is a little different, it's git + ruby.  This separates concerns in a way I find aesthetically appealing - if we break the terraform generation container, we don't also have to break puppet forge pushes.